pkg/util/limiter/utilization.go

```diff
@@ -79,6 +83,22 @@ func NewUtilizationBasedLimiter(cpuLimit float64, memoryLimit uint64, logger log
 		cpuMovingAvg: math.NewEWMARate(alpha, resourceUtilizationUpdateInterval),
 	}
 	l.Service = services.NewTimerService(resourceUtilizationUpdateInterval, l.starting, l.update, nil)
+
+	if reg != nil {
+		promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "utilization_limiter_current_cpu_load",
+			Help: "Current average CPU load calculated by utilization based limiter.",
+		}, func() float64 {
+			return l.currCPUUtil.Load()
+		})
+		promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "utilization_limiter_current_memory_utilization",
```

Comment:
[nit] By [convention](https://prometheus.io/docs/practices/naming/), should end with "bytes".

```suggestion
			Name: "utilization_limiter_current_memory_usage_bytes",
```

Reply:
Thanks!